packages/wxt/src/core/runners/web-ext.ts

```diff
@@ -81,6 +82,7 @@ export function createWebExtRunner(): ExtensionRunner {
 
       const duration = Date.now() - startTime;
       wxt.logger.success(`Opened browser in ${formatDuration(duration)}`);
+      wxt.logger.info(`To reopen the browser, press ${pc.bold('o + enter')}`);
```

Comment:
```suggestion
      wxt.logger.info(`Reopen browser using ${pc.bold('o')} + ${pc.bold('ENTER')}`);
```

Reply:
Tested, works fine for me on mac with CTRL + C too, I do have this one change just to make the text look a bit better. @aklinker1 thoughts?

Reply:
Looks good, is that the same wording and casing Vite uses? We should just match what they do... Away from my computer, can't check.

Reply:
![image](https://github.com/user-attachments/assets/006c03c5-b1c5-4cb8-9beb-4bf089efc0a4)

~~Well if you want it to look like vite it has a bit of work to go jk~~

Reply:
![image](https://github.com/user-attachments/assets/01cdb36d-8665-4f5c-bf34-16da5c707bd7)
Ah, that's what it looks like. I think this should be fine for now, once we add more tools and stuff we can make the ui look better? Should I change my suggestion and we stick with it @aklinker1 or do we keep it like I suggested it for now?